extensions/amp-consent/0.1/amp-consent.js

```diff
@@ -611,7 +611,14 @@ export class AmpConsent extends AMP.BaseElement {
                 .fetchJson(expandedHref, init)
                 .then((res) =>
                   xhrService.xssiJson(res, this.consentConfig_['xssiPrefix'])
-                );
+                )
+                .catch((e) => {
+                  user().error(
+                    TAG,
+                    'Could not parse the `checkConsentHref` response.',
```

Comment:
We still need to throw an error because we don't want to proceed with an invalid response. Fallback to empty response is not desired because empty response equals to consent not required. 
 
Also since the error is strictly related to xssiJson, I'd do
```
try {
  xhrService.xssiJson()...
} catch(e) {
  userAssert(false, .....)
}
```